### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by rudy3333**
the change of nightshadetvn -> rudy3333, is because it's my dead name.

### CODE REVIEW

**Comment by TheLastProject** [comment]
Thanks and welcome back :)

File: README.md [hunk 1]
```diff
@@ -43,3 +43,4 @@ Breaking the rules WILL end in your PR being marked as spam. This could cause yo
 I did 34 translations/changes inside Catima glossary and android component for Hindi language.
 - 01-10-2023 [@ziadOUA](https://github.com/ziadOUA) Translated the remaining Android strings in Italian
 - 04-10-2023 https://hosted.weblate.org/translate/catima/catima/hi/?checksum=07f7cfb37b087d85  Translation for Hindi Language in Android.
+- 04-10-2023 [@rudy3333](https://github.com/rudy3333) Added ~100 Polish translations (and still counting!) to Caitma Android Fastlane.
```
Comment (obfusk) [inline README.md:46]: might want to fix the "Caitma" typo :sweat_smile:

Comment (TheLastProject) [reply README.md:46]: Eh, it's fine, this repo only exists to track Hacktoberfest contributions. It's bookkeeping, nothing else :)

Comment (rudy3333) [reply README.md:46]: 😭😁sorry lolz it was 1am for me

**Pull request merged**